### DISCUSSION TITLE
"item stats" plugin show tooltip only in bank view

### DISCUSSION THREAD

**Discussion by AnthonyPoschen** [runelite/runelite#17595] [Ideas]
typically i never need to see the stats while bossing / raiding. I only ever need it when trying to figure out setups in the bank.

could an option be added to "show Stats outside Bank" that is on by default added under this entry?
which would allow me to have the plugin disabled except for while in a bank view.

![image](https://github.com/runelite/runelite/assets/2253299/ad2582de-7050-42f9-8fe3-1ea219c3ebc1)

A few PR's have been raised for things like showing it in the inventory screen etc which could follow the same pattern as this if all ideas are accepted.

**Comment by zeroquinc** [comment]
+1, surprised this is not a thing yet.

**Comment by fotuoh** [comment]
+1, I honestly thought this was already implemented. Mandela Effect?

**Comment by TheEcte** [comment]
so... when is this going to be added?

**Reply by coopermor** [reply]
Feel free to add the feature in a pull request.
